lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -32,27 +32,8 @@ def import_module(name):  # type: ignore[misc]
         __import__(name)
         return sys.modules[name]
 
-try:
```

Comment:
This Python module needs to support target Python versions, not just controller Python versions. So the Python 2.7 support can be dropped, but not Python 3.8.

Reply:
On a related note, there is more Python 2.7 code that can be removed, such as the use of `ansible.module_utils.six`.

Reply:
We're not removing that feature -- we're just discontinuing use of it in core where it's no longer needed.

Reply:
That's totally sensible :) Thanks for clarifying!